immuni_analytics/celery/scheduled/tasks/store_operational_info.py

```diff
@@ -0,0 +1,57 @@
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+#   Please refer to the AUTHORS file for more information.
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#   GNU Affero General Public License for more details.
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program. If not, see <https://www.gnu.org/licenses/>.
+import asyncio
+import json
+import logging
+from typing import Any, Dict
+
+from immuni_analytics.celery.scheduled.app import celery_app
+from immuni_analytics.core import config
+from immuni_analytics.core.managers import managers
+from immuni_analytics.models.operational_info import OperationalInfo
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@celery_app.task()
+def store_operational_info() -> None:  # pragma: no cover
+    """
+    Celery doesn't support async functions, so we wrap it around asyncio.run.
+    """
+    asyncio.run(_store_operational_info())
```

Comment:
Maybe worth creating an `async_celery_task` decorator which takes the `celery_app` as an argument?

Then we'd have something like:
```python
@async_celery_task(celery_app)
async def _store_operational_info() -> None:
    """
    Retrieve up to a fixed number of operational info and save it into mongo.
    """

    _LOGGER.info("Store operational info periodic task started.")
```

Reply:
I would tackle this in another pr.